template_header.tex

```diff
@@ -12,7 +12,7 @@
 \usepackage{multicol}
 \usepackage{graphicx}
 
-\geometry{verbose,landscape,${paper},tmargin=2cm,bmargin=2cm,lmargin=1cm,rmargin=1cm}
+\geometry{verbose,landscape,${paper},tmargin=0.3cm,bmargin=0.3cm,lmargin=0.5cm,rmargin=0.5cm}
```

Comment:
It seems interesting to pass this as a CLI parameter (defined in bin/notebookgen). I think it might help in case people have issues when printing.